client/app/scripts/liveblog-edit/timeline.js

```diff
@@ -197,12 +197,42 @@ define([
             }
         };
     }])
-    .directive('itemActions', ['api', 'notify', 'gettext', 'asset', 'itemsService', 'modal',
+    .directive('postActions', ['api', 'notify', 'gettext', 'asset', 'itemsService', 'modal',
         function(api, notify, gettext, asset, itemsService, modal) {
         return {
             restrict: 'E',
             replace: true,
-            templateUrl: 'scripts/liveblog-edit/views/timeline-item-actions.html',
+            templateUrl: 'scripts/liveblog-edit/views/timeline-post-actions.html',
+            link: function(scope, elem, attrs) {
```

Comment:
It would be maybe better to use an isolated scope for directives. It prevents to erase properties and make clear what is available in the scope.

It's not only me who tells it :)
From the [angular doc](https://docs.angularjs.org/guide/directive)
> Best Practice: Use the scope option to create isolate scopes when making components that you want to reuse throughout your app.